landoapi/api/revisions.py

```diff
@@ -53,6 +62,29 @@ def _format_revision(
     Returns:
         A hash of the formatted revision information.
     """
+    # Load the revision's diff if requested.
+    diff = None
+    if include_diff and revision['activeDiffPHID']:
+        raw_diff = phab.get_diff(phid=revision['activeDiffPHID'])
+        diff = {
+            'id': int(raw_diff['id']),
+            'revision_id': int(raw_diff['revisionID']),
+            'date_created': int(raw_diff['dateCreated']),
+            'date_modified': int(raw_diff['dateModified']),
+            'vcs_base_revision': raw_diff['sourceControlBaseRevision'],
+            'authors': None
+        }
+
+        if raw_diff['properties'] and raw_diff['properties']['local:commits']:
```

Comment:
This is by design, 'properties' is always present but can be an empty dict or None, and so for 'local:commits'.